PollBuddy-Server/frontend/src/pages/GroupJoin/GroupJoin.js

```diff
@@ -0,0 +1,65 @@
+import React, { Component } from "react";
+import "mdbreact/dist/css/mdb.css";
+import { MDBContainer } from "mdbreact";
+
+export default class GroupJoin extends Component {//this class will likely need to call Groups/new and do more with that...
+  componentDidMount(){
+    this.props.updateTitle("Group Join");
+  }
+
+  constructor() {
+    super();
+    //TODO check if they're logged in
+    console.log(localStorage.getItem("loggedIn"));
+    if (localStorage.getItem("loggedIn") == "true") {
```

Comment:
This is fine for now but we may rework that later which could solve the sporadic issues you've been having.